src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -104,6 +114,8 @@ class SidebarPanels extends React.Component<Props, State> {
 
     metadataSidebar: ElementRefType = React.createRef();
 
+    metadataSidebarRedesigned: ElementRefType = React.createRef();
```

Comment:
since only one version of metadatasidebar can render at a time, you could reuse the ref above for the new redesign so there's less code to clean up

Reply:
Good idea, I delete it